assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java

```diff
@@ -677,10 +677,10 @@ public ActionReturn getGradableForSite(EntityView view , Map<String, Object> par
             Map<String, Object> content = ltiService.getContent(contentKey.longValue(), site.getId());
             String contentItem = StringUtils.trimToEmpty((String) content.get(LTIService.LTI_CONTENTITEM));
 
-            for (SimpleSubmission submission : submissions) {
+            for ( SimpleSubmission submission : submissions ) {
```

Comment:
avoid these type of ws changes

Reply:
I was copying the Chuck method. I'll remove it.